aws_fuzzy_finder/settings.py

```diff
@@ -13,10 +13,9 @@
 ENV_SSH_COMMAND_TEMPLATE = os.getenv('AWS_FUZZ_SSH_COMMAND_TEMPLATE', "ssh {key} {user}{host}")
 CACHE_EXPIRY_TIME = int(os.getenv('AWS_FUZZ_CACHE_EXPIRY', 3600))
 CACHE_ENABLED = os.getenv('AWS_FUZZ_USE_CACHE', False)
-CACHE_PATH = '{}/{}'.format(
-    expanduser("~"),
-    '.aws_fuzzy_finder.cache'
-)
+AWS_DEFAULT_PROFILE=os.getenv('AWS_DEFAULT_PROFILE', '_noprofile')
+CACHE_DIR = '{}/{}'.format(expanduser("~"), '.aws_fuzzy_finder.cache')
+CACHE_PATH = '{}/{}'.format(CACHE_DIR, AWS_DEFAULT_PROFILE)
```

Comment:
It should use `AWS_PROFILE` IMHO. Tools such as boto and node are standardizing on using AWS_PROFILE instead `AWS_DEFAULT_PROFILE`.

Besides that, good patch! I have been used it.